CHANGELOG.md

```diff
@@ -88,6 +113,10 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 - Changed method to perform redirection on agent table buttons [#5539](https://github.com/wazuh/wazuh-kibana-app/pull/5539)
 - Changed windows agent service name in the deploy agent wizard [#5538](https://github.com/wazuh/wazuh-kibana-app/pull/5538)
+  <<<<<<< HEAD
+  =======
```

Comment:
**nitpick**: I think the conflict was not correctly solved.

Reply:
Done